common/device-manager.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * DeviceManager Stub
+ * TODO(cjdaly) integrate with Mock Local Home Platform and Mock Network
+ **/
+/// <reference types="@google/local-home-sdk" />
+
+export const DeviceManagerStub = class {
+  deviceId: string;
+  error: boolean;
+  public commands = new Array<smarthome.DataFlow.TcpRequestData>();
+
+  public send(command: smarthome.DataFlow.TcpRequestData): Promise<any> {
```

Comment:
Is there a better return type than `any`?

Reply:
Yeah, I think it should be Promise<DataFlow.CommandSuccess> to match the sdk.  Will change